src/components/RotatingImage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Image } from "react-native";
+import { GestureDetector, Gesture } from "react-native-gesture-handler";
+import {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from "react-native-reanimated";
+import Animated, { Easing } from "react-native-reanimated";
+
+const duration = 2000;
+const easing = Easing.bezier(0.25, -0.5, 0.25, 1);
+
+const RotatingImage = ({ imageUri }: { imageUri?: string }) => {
+  const sv = useSharedValue(0);
```

Comment:
You can use `v` as variable name, so it will be even shorter!
```suggestion
  const v = useSharedValue(0);
```

Reply:
so an ironic comment!